.vuepress/style.styl

```diff
@@ -44,6 +44,7 @@
 
 .sidebar
     background-color #f7f7f7 !important
+    z-index: 15
```

Comment:
Minor nitpicking: can you remove the : here for consistency (it's optional in Stylus).

Reply:
This is fixed :)!